src/components/screens/ScoreSetEditor.vue

```diff
@@ -299,23 +298,29 @@
                         </div>
                       </div>
                       <div class="field">
-                        <span class="p-float-label">
-                          <Dropdown v-model="referenceGenome" :id="$scopedId('input-targetGeneReferenceGenome')"
-                            :options="referenceGenomes" panelClass="mave-reference-genome-dropdown-panel">
-                            <template #value="slotProps">
-                              <div v-if="slotProps.value" class="mave-reference-genome-value">
-                                <div class="mave-reference-genome-name">{{ slotProps.value.shortName }}</div>
-                                <div class="mave-reference-genome-organism-name">{{ slotProps.value.organismName }}</div>
-                              </div>
-                              <div v-else class="mave-reference-genome-none">&nbsp;</div>
-                            </template>
-                            <template #option="slotProps">
-                              <div class="mave-reference-genome-name">{{ slotProps.option.shortName }}</div>
-                              <div class="mave-reference-genome-organism-name">{{ slotProps.option.organismName }}</div>
-                            </template>
-                          </Dropdown>
-                          <label :for="$scopedId('input-targetGeneReferenceGenome')">Reference genome</label>
-                        </span>
+                        If taxonomy does not appear on the option list, please enter a taxonomy ID, organism name or common name from 
```

Comment:
is this text still necessary? it looks from the backend code like autocomplete searches both the local database and the ncbi database and provide both as options without the user hitting enter

Reply:
No, it's better to delete it.